### PR TITLE
Persons for b lorient547

### PR DESCRIPTION
**Pull request by MarcinKrawczuk**
So these are five people attested in one ms. Hensa Selassie is father of Walatta Takla Haymanot and Haile Takla Haymanot. The other two are separately. The floruit is based on the date for the manuscript.

### CODE REVIEW

File: new/PRS13410Walatta.xml [hunk 1]
```diff
@@ -0,0 +1,64 @@
...
+        </profileDesc>
+        <revisionDesc>
+            <change who="MKr" when="2021-05-11">Created entity</change>
+        </revisionDesc>
+    </teiHeader>
+    <text>
+        <body>
+            <listPerson>
+                <person sex="2">
+                    <persName xml:lang="gez" xml:id="n1">ወለተ፡ ትክለ፡ ሃይማኖት፡</persName>
```
Comment (DariaElagina) [inline new/PRS13410Walatta.xml:48]: ተክለ፡

File: new/PRS13414Walatta_.xml [hunk 2]
```diff
@@ -0,0 +1,63 @@
+<?xml-model href="https://raw.githubusercontent.com/BetaMasaheft/schema/master/tei-betamesaheft.rng"
+schematypens="http://relaxng.org/ns/structure/1.0"?><?xml-model href="https://raw.githubusercontent.com/BetaMasaheft/schema/master/tei-betamesaheft.rng"
+type="application/xml" schematypens="http://purl.oclc.org/dsdl/schematron"?>
+<TEI xmlns="http://www.tei-c.org/ns/1.0" xml:id="PRS13414Walatta_" xml:lang="en" type="pers">
```
Comment (DariaElagina) [inline new/PRS13414Walatta_.xml:4]: An Id with _ is not ok: https://betamasaheft.eu/Guidelines/?id=entities-id-structure
Please, ask Pietro what would be the best way to correct it, if you can just do it on this branch changing the id and the name of the file.

**Review by DariaElagina — CHANGES_REQUESTED** [review]

**Review by PietroLiuzzo — APPROVED** [review]

File: new/PRS13414Walatta.xml [hunk 3]
```diff
@@ -0,0 +1,63 @@
...
+                    <persName xml:lang="gez" xml:id="n1">ወለተ፡ ጊዮርጊስ፡</persName>
+                    <persName xml:lang="gez" type="normalized" corresp="#n1">Walatta Giyorgis</persName>
+                    <faith type="EOTC"/>
+                    <nationality type="Ethiopia"/>
+                    <birth/>
+                    <death/>
+                    <floruit notBefore="1750" notAfter="1800"/>
+                </person>
+                <listRelation>
+                    <relation name="lawd:hasAttestation" active="PRS13414Walatta_" passive="BLorient547"/>
```
Comment (DariaElagina) [inline new/PRS13414Walatta.xml:57]: Here the id is with _

**Review by DariaElagina — CHANGES_REQUESTED** [review]

File: new/PRS13414Walatta.xml (hunk 3 above)
Comment (PietroLiuzzo) [reply new/PRS13414Walatta.xml]: I would dare to suggest that such changes are faster done than reported :) !

**Review by DariaElagina — APPROVED** [review]

**Pull request merged**